src/HttpHdrSc.cc

```diff
@@ -108,7 +108,7 @@ HttpHdrSc::parse(const String * str)
         type = scLookupTable.lookup(SBuf(item,ilen));
 
         if (type == SC_ENUM_END) {
-            debugs(90, 2, "hdr sc: unknown control-directive: near '" << item << "' in '" << str << "'");
+            debugs(90, 2, "hdr sc: unknown control-directive: near '" << item << "' in '" << *str << "'");
```

Comment:
Just polishing to improve phrasing and remove duplicated information from this modified debugs() line. The removed information is added automatically by debugs() to all level-2+ messages. This will also make this line a bit more consistent with HttpHdrRange debugging:

```suggestion
            debugs(90, 2, "unknown control-directive near '" << item << "' in '" << *str << "'");
```